backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -61,12 +61,14 @@ public TestEventExport(TestEvent testEvent) {
           "ambiguous", "A",
           "unknown", genderUnknown,
           "unk", genderUnknown,
+          "refused", genderUnknown,
```

Comment:
Made some assumptions in this file that "refused" would generally map to "unknown" since there aren't really specific options for refused in the allowed data